### PR TITLE
TM-4707: Job categories integration

### PR DESCRIPTION
**Pull request by pveland621**
This PR covers the functionality of loading in data for the job categories dropdown and also loading all skills and checking the box if the skill is associated with the selected job category.

Follow up PR will address the ability to save edits to the skills associated with a job category

Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2693/files)

[Ticket](https://metaphase.atlassian.net/browse/TM-4707)

### CODE REVIEW

File: talentmap_api/fsbid/services/job_categories.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+#         }
+
+#     return {
+#         'bureauFilters': list(map(bureau_map, data.get('PQRY_BUREAU_LEVEL_O'))),
+#         'personFilters': list(map(person_map, data.get('PQRY_PERSON_LEVEL_O'))),
+#         'roleFilters': list(map(role_map, data.get('PQRY_POST_ROLE_O'))),
+#         'positionFilters': list(map(position_map, data.get('PQRY_POSITION_LEVEL_O'))),
+#         'locationFilters': list(map(location_map, data.get('PQRY_COUNTRY_O'))),
+#         'orgFilters': list(map(org_map, data.get('PQRY_ORG_LEVEL_O'))),
+#     }
```
Comment (M00staff) [inline talentmap_api/fsbid/services/job_categories.py:75]: so, I'm all for making a cleanup PR afterwards to clean up logging statements and get this out the door - but does THIS need to be here?

File: talentmap_api/fsbid/services/common.py [hunk 2]
```diff
@@ -306,15 +306,13 @@ def send_post_back_office(proc_name, package_name, request_body, request_mapping
     url = f"{BACKOFFICE_CRUD_URL}?procName={proc_name}&packageName={package_name}"
     json_body = request_mapping_function(request_body)
     response = requests.post(url, headers={'JWTAuthorization': jwt_token, 'Content-Type': 'application/json'}, json=json_body).json()
-    if response is None or (response['PV_RETURN_CODE_O'] and response['PV_RETURN_CODE_O'] is not 0):
-        logger.error(f"Fsbid call to '{url}' failed.")
-        return None
+    # if response is None or (response['PV_RETURN_CODE_O'] and response['PV_RETURN_CODE_O'] is not 0):
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/common.py:309]: Can you please leave a comment for why you did this - like how the error was presenting, so we can go address later

Comment (scott062) [reply talentmap_api/fsbid/services/common.py:309]: Leave it on the PR for now

Comment (pveland621) [reply talentmap_api/fsbid/services/common.py:309]: yes sorry - talked with moose about this. the payloads do not have the same return code key names (`PV_RETURN_CODE_O` for moose's, i think mine is something along the lines of `RETURN_CODE_O`), will fix this in followup PR but this shouldnt break any functionality

**Pull request merged**